### DISCUSSION TITLE
Using CCD vs covalent bond with smiles

### DISCUSSION THREAD

**Discussion by kanghw0325** [chaidiscovery/chai-lab#268] [Announcements]
Hello! Thank you for the great work.

I am recently working on your covalent bond system.
and I see one small issues in here.

In my opinion, using SMILES seems to yield better results than using CCD when forming covalent bonds. I’m not sure why.
Do you have any informations of this kind of issues?

**Comment by arogozhnikov** [comment]
I recommend checking if it tries to match reference conformer for CCD when you use CCD.

**Reply by kanghw0325** [reply]
sorry for late reply, ccd is here.

CCD: 99K
SMILES: [O-][N+](=O)c1cccnc1Nc2ncc(cc2Cl)C(F)(F)F

i think smiles is better than CCD
